client/index.html

```diff
@@ -175,6 +180,7 @@ <h1 class="title">Connect</h1>
 				<div id="settings" class="window">
 					<div class="header">
 						<button class="lt" aria-label="Toggle channel list"></button>
+						<span class="title">Settings</span>
```

Comment:
This (and the previous one) just feels odd. It adds nothing to see "Settings" right above "Settings". If the header was static, maybe (though really not sold), but here... Meh.